packages/tasks/src/model-data.ts

```diff
@@ -107,7 +107,7 @@ export interface ModelData {
 					parameters?: Record<string, unknown>;
 			  };
 		base_model?: string | string[];
-		instance_prompt?: string;
+		instance_prompt?: string | null;
```

Comment:
Interesting, I would have thought that the trailing `?` would imply that.

Reply:
`instance_prompt?: string` is `string | undefined`
`instance_prompt?: string | null` is `string | null | undefined`

in the strictest sense `null !== undefined`

Reply:
TIL

Reply:
ok when can it be set to `null`?

Reply:
I'm not sure but that is how it was defined in moon:

`instance_prompt?: string | null;`